build/configs/artik053/STDK/defconfig

```diff
@@ -29,7 +29,6 @@ CONFIG_BUILD_FLAT=y
 #
 # CONFIG_INTELHEX_BINARY is not set
 # CONFIG_MOTOROLA_SREC is not set
-CONFIG_RAW_BINARY=y
```

Comment:
You removed this in all of defconfigs. But I can't find Kconfig change related this. Could you check it?

Reply:
Yes. I had missed the kconfig. fixing it.

Reply:
Sorry for late suggestion.
RAW binary is mandatory on TizenRT but in output extension on menuconfig does not have it. And there is no content for this. To know it easily, how about keeping this config, hidden it from menuconfig and set it default y?
How do you think? Then we don't need to modify all of defconfigs.

Reply:
By default all of the defconfig has the binary as y. However it is still configurable. So unknowingly if someone changes it. It will cause build failures. So i think it is better to remove the configs itself so no configuration is possible by menuconfig or by force by updating the defconfig. Just to be on safe side.

Reply:
That's why I suggest hiding it form menuconfig.
```
config AAAAA
    bool
```
If you remove "description" after type, menuconfig does not show it up. So user can't disable it.

Reply:
```
config AAA
   bool "description"
```
User can control this on the menuconfig (on / off).
```
config AAA
   bool
   default y
```
This is enabled always and user can't disable it.